test/common.sh

```diff
@@ -4,3 +4,19 @@ set -eu -o pipefail
 
 ROOT="$DIR/../"
 INSTALLER="$ROOT/script/install.sh"
+
+assert() {
+    actual="$1"
+    expected="$2"
+    if [ "$actual" != "$expected" ] ; then
+        echo "Actual: $actual"
+        echo "Expected: $expected"
+        exit 1
+    fi
```

Comment:
For this I would simply write `diff -u <(cat <<< "$1") <(cat <<< "$2")`, but this is OK too.